python/pyspark/pandas/plot/matplotlib.py

```diff
@@ -375,6 +398,7 @@ def _make_plot(self):
             kwds = self.kwds.copy()
 
             label = pprint_thing(label if len(label) > 1 else label[0])
+            label = self._mark_right_label(label, index=i)
```

Comment:
How about this approach? I think this might be simpler.

```suggestion
            # Makes plotting compatible with pandas < 1.3
            label = self._mark_right_label(
                label, index=i) if hasattr(self, "_mark_right_label") else label
```

Reply:
Good idea.

After "some time" the part `if hasattr(...) else label` can be just removed.